bindata/network/multus/multus.yaml

```diff
@@ -132,6 +132,11 @@ spec:
         - "--multus-conf-file=auto"
         - "--multus-autoconfig-dir=/host/var/run/multus/cni/net.d"
         - "--multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig"
+{{- if eq .DefaultNetworkType "OpenShiftSDN"}}
+        - "--readiness-indicator-file=/var/run/multus/cni/net.d/80-openshift-network.conf"
+{{- else if eq .DefaultNetworkType "OVNKubernetes"}}
+        - "--readiness-indicator-file=/var/run/multus/cni/net.d/10-ovn-kubernetes.conf"
+{{- end}}
```

Comment:
Do we need kuryr as well?

Reply:
Oh, I meant to comment on that; the file kuryr uses isn't mentioned anywhere in CNO, it's only referenced from the kuryr module. So it didn't seem right to add a dependency on it here since they might change the name in the future.

If `--readiness-indicator-file` isn't set then multus just falls back to its old behavior from before this was added, so nothing *bad* happens, it's just that nothing extra-good happens either.

Reply:
OK